draft-rokui-ccamp-actn-wdm-pluggable-modelling.txt

```diff
@@ -123,53 +103,56 @@ Internet-Draft        Modelling Optical Pluggables         November 2024
    described in Section 4.e of the Trust Legal Provisions and are
    provided without warranty as described in the Revised BSD License.
 
+
+
+
+
+
+
+Rokui, et al.             Expires 18 June 2025                  [Page 2]
+
+Internet-Draft        Modelling Optical Pluggables         December 2024
+
+
 Table of Contents
 
    1.  Terminology . . . . . . . . . . . . . . . . . . . . . . . . .   4
    2.  Introduction  . . . . . . . . . . . . . . . . . . . . . . . .   5
    3.  Optical Pluggable in a Device with Packet Functions . . . . .   7
    4.  Coherent Pluggable Functional Building Blocks . . . . . . . .   9
      4.1.  Optical Channel/OTSI  . . . . . . . . . . . . . . . . . .  10
-     4.2.  Media Logical Channels  . . . . . . . . . . . . . . . . .  10
-     4.3.  Host Logical Channels . . . . . . . . . . . . . . . . . .  10
+     4.2.  Media Logical Channels  . . . . . . . . . . . . . . . . .  11
+     4.3.  Host Logical Channels . . . . . . . . . . . . . . . . . .  11
      4.4.  Electrical Channels . . . . . . . . . . . . . . . . . . .  11
-     4.5.  Equipment . . . . . . . . . . . . . . . . . . . . . . . .  11
+     4.5.  Equipment . . . . . . . . . . . . . . . . . . . . . . . .  12
    5.  Optical Pluggables Data Modeling  . . . . . . . . . . . . . .  12
      5.1.  Coherent Pluggable Capability Attributes (aka,
-           Supported-Modes)  . . . . . . . . . . . . . . . . . . . .  12
-     5.2.  Coherent Pluggable Configurations Attributes  . . . . . .  13
+           Supported-Modes)  . . . . . . . . . . . . . . . . . . . .  13
+     5.2.  Coherent Pluggable Configurations Attributes  . . . . . .  14
      5.3.  Coherent Pluggable Performance Monitoring Data  . . . . .  14
      5.4.  Coherent Pluggable Threshold Definition . . . . . . . . .  16
-     5.5.  Coherent Pluggable Alarm Notifications  . . . . . . . . .  17
-     5.6.  Support of Opaque Attributes  . . . . . . . . . . . . . .  17
-       5.6.1.  Support of Opaque Capability Attributes . . . . . . .  19
-       5.6.2.  Support of Opaque Secret Capability Attributes  . . .  19
+     5.5.  Coherent Pluggable Alarm Notifications  . . . . . . . . .  18
+     5.6.  Support of Opaque Attributes  . . . . . . . . . . . . . .  18
```

Comment:
I doubt that opaque attributes are needed! Each vendor can augment the model with vendor-specific attributes.

Reply:
OK - let's discuss this when we address Open Issue #14 .